app/assets/javascripts/utils/api.js

```diff
@@ -503,15 +503,12 @@ slide_id=${opts.slide_id}`,
     return new Promise((res, rej) =>
       $.ajax({
         type: 'GET',
-        url: `/courses/${courseId}/needs_update.json`,
-        success(data) {
-          alert(data.result);
-          return res(data);
-        }
+        url: `/courses/${courseId}/needs_update.json`
       })
+      .done(obj => res(obj))
```

Comment:
You can do this in the 'success' call to match the pattern through most of api.js.